dashboard/src/containers/AccessURLTableContainer/AccessURLTableContainer.tsx

```diff
@@ -22,10 +19,4 @@ function mapStateToProps({ kube }: IStoreState, props: IAccessURLTableContainerP
   };
 }
```

Comment:
Not necessarily in this PR, but we should also get rid of the xxxxContainer.tsx in favor of just using react hooks, especially, now we have removed more logic